src/components/navBar/navBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./navBar.css";
+const navBar = () => {
```

Comment:
- React user defined components should be capitalized or else react can mistake your components for html tags and other nasty bugs.
**Reference** - https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized